charts/capsule/crds/capsule.clastix.io_tenants.yaml

```diff
@@ -1149,6 +1149,12 @@ spec:
                 description: Toggling the Tenant resources cordoning, when enable
                   resources cannot be deleted.
                 type: boolean
+              forceTenantPrefix:
```

Comment:
I am not yet happy with the description of the CRD. Please make it more understandable for endusers

Also you need to generate the new api manifests via `make generate` and `make manifests`

Reply:
How about:
```yaml
              forceTenantPrefix:
                default: false
                description: |-
                  Use this if you want to enforce Tenant name prefix to specific Tenants instead of globally with forceTenantPrefix in CapsuleConfiguration.
                  Enforces Namespace created for this Tenant to be named with the Tenant name prefix,
                  separated by a dash (i.e. for Tenant 'foo', namespace names must be prefixed with 'foo-').
                  This is useful to avoid Namespace name collision.
```